js/web/lib/onnxjs/attribute-with-cache-key.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+class AttributeWithCacheKeyImpl {
+  constructor(attribute: Record<string, unknown>) {
+    Object.assign(this, attribute);
+  }
+
+  private _cacheKey: string;
+  public get cacheKey(): string {
+    if (!this._cacheKey) {
+      this._cacheKey =
+          Object.getOwnPropertyNames(this).sort().map(name => `${(this as Record<string, unknown>)[name]}`).join(';');
```

Comment:
Is there any case that the order of names returned by getOwnPropertyNames() is not consistent? If it's consistent, is sort() required here? It may affect an initialization performance.

Reply:
`getOwnPropertyNames()` returns the keys in the order of insertion. we don't ensure that when every time we construct the attribute object, the keys are inserted in the same order. it's safe to do a sort. currently the biggest attribute object contains around 10 keys so this cost is trivial.